go.mod

```diff
@@ -6,7 +6,7 @@ toolchain go1.25.1
 
 require (
 	github.com/google/uuid v1.6.0
-	github.com/kubewarden/kubewarden-controller v1.29.0
+	github.com/kubewarden/kubewarden-controller v1.29.0-rc3
```

Comment:
Notice, renovate bot [removed](https://github.com/kubewarden/audit-scanner/commit/69afa4ac4faaca945f1f4d01e1293cff1e3ec473#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R9) the previous `rc2` suffix.